version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "9.0.1-SNAPSHOT"
```

Comment:
The version.sbt file should never be changed. The `sbt release` command will update it in a proper way